INSTALL.md

```diff
@@ -443,7 +446,7 @@ build QGIS.
 
 The free (as in free beer) Community installer is available under:
 
-> http://download.microsoft.com/download/D/2/3/D23F4D0F-BA2D-4600-8725-6CCECEA05196/vs_community_ENU.exe
+https://download.microsoft.com/download/D/2/3/D23F4D0F-BA2D-4600-8725-6CCECEA05196/vs_community_ENU.exe
```

Comment:
Is it a writing convention to not provide the URL as part of the ongoing paragraph but as a new one?

Reply:
I'd actually rewrite to.

Download the [free (as in beer) Community installer](https://download.microsoft.com/download/D/2/3/D23F4D0F-BA2D-4600-8725-6CCECEA05196/vs_community_ENU.exe).